docs/ml/_index.md

```diff
@@ -16,19 +16,42 @@ menuindent: true
 {{<imgproc src="/ml/training.png" class="alignright" resize="400x" declaredimensions=true alt="ML training">}}
 
 Machine Learning (ML) provides your machines with the ability to adjust its behavior based on models that recognize patterns or make predictions.
+
 Common use cases include:
 
-- Object detection and classification which enable machines to detect people, animals, plants, or other objects with bounding boxes, and to perform actions when they are detected.
+- Object detection, which enables machines to detect people, animals, plants, or other objects with bounding boxes, and to perform actions when they are detected.
+- Object classification, which enables machines to separate people, animals, plants, or other objects into predefined categories based on their characteristics.
 - Speech recognition, natural language processing, and speech synthesis, which enable machines to verbally communicate with us.
 
 However, your machine can make use of machine learning with nearly any kind of data.
 
-Viam supports:
+{{% alert title="Note" color="note" %}}
+See [Integrate Viam with ChatGPT to Create a Companion Robot](/tutorials/projects/integrating-viam-with-openai/) for an example of using Viam to add object detection, speech recognition, natural language processing, and speech synthesis capabilities to a machine.
+{{% /alert %}}
+
+Viam provides models of [ML model](/ml/deploy/) and [Computer Vision](/ml/vision/) services to facilitate model deployment and object detection and classification.
+
+The ML model service runs the model on your machine so that the [Computer Vision](/ml/vision) `mlmodel` service can use it as a detector or classifier, interpreting image data from images on your computer or a [camera](/components/camera/) to draw bounding boxes around objects (detection) or return a class label and confidence score (classification) based off the [inferences](/ml/deploy/#infer) the underlying ML model makes.
```

Comment:
```suggestion
The ML model service runs the model on your machine so that the [Computer Vision](/ml/vision/) `mlmodel` service can use it as a detector or classifier, interpreting image data from images on your computer or a [camera](/components/camera/) to draw bounding boxes around objects (detection) or return a class label and confidence score (classification) based off the [inferences](/ml/deploy/#infer) the underlying ML model makes.
```